client-base/src/test/java/app/cash/backfila/embedded/CaseParameters.java

```diff
@@ -9,14 +10,21 @@ public class CaseParameters {
   public final long testLong;
   public final int testInt;
   public final boolean testBool;
+  public final String testNullString;
+  public final Integer testNullInt;
+  public final Boolean testNullBoolean;
+  public final String required;
 
   public CaseParameters(
       @Description(text = "Whether to change case to upper case or lower case.")
       @BackfilaDefault(name = "casing", value = "upper") String casing,
       @BackfilaDefault(name = "testLong", value = "123") long testLong,
       @BackfilaDefault(name = "testInt", value = "789") int testInt,
       @BackfilaDefault(name = "testBool", value = "false") boolean testBool,
-      @BackfilaRequired(name = "required") String requiredParameter) {
+      @BackfilaNullDefault(name = "testNullString") String testNullString,
```

Comment:
Should we enforce `@Nullable` on these types of parameters?

Reply:
Or rather... instead of introducing a new annotation, can we implement the same behaviour when a parameter is annotated like `@BackfilaDefault(name = "foo") @Nullable String foo`?

Then we complain if no value is set, and the parameter isn't annotated as nullable?

Reply:
Never mind I guess I was reviewing an old diff!